specs/windows/nt_info.table

```diff
@@ -0,0 +1,23 @@
+table_name("nt_info") description(
+    "Display basic NT domain information of a Windows machine.")
+    schema([
+      Column("name", TEXT, "The label by which the object is known."),
```

Comment:
As far as it is a `label` according to the description would it be better to use the same word for the column name?